app/src/main/java/com/example/notesmvvm/AdapterClass.kt

```diff
@@ -9,29 +9,33 @@ import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.LinearLayout
 import android.widget.TextView
+import androidx.databinding.DataBindingUtil
 import androidx.recyclerview.widget.RecyclerView
 import com.example.notesmvvm.Fragments.AddEdit
+import com.example.notesmvvm.databinding.ActivityMainBinding
+import com.example.notesmvvm.databinding.FragmentAddEditBinding
+import com.example.notesmvvm.databinding.FragmentAllNotesBinding
+import com.example.notesmvvm.databinding.ItemSampleBinding
 
 class AdapterClass(private val context : Context, val listener : NotesAdapter, val editlistener : NotesAdapter) : RecyclerView.Adapter<AdapterClass.viewHolder>() {
```

Comment:
use ListAdapter ( not ListView Adapter) with diffUtils instead of recyclerView adapter